hash/x68k_flop.xml

```diff
@@ -2984,13 +3004,47 @@ Most info on release dates and Jpn titles come from the following (wonderful) re
 		</part>
 	</software>
 
-	<software name="cntinent">
+	<!-- Hangs after the Game Technopolis logo -->
```

Comment:
Please push this comment to an actual `notes` node.

Reply:
Done.